### PR TITLE
update copier

### PR DESCRIPTION
**Pull request by jokasimr**
Only for testing - should not be merged.

### CODE REVIEW

**Review by jl-wynen — APPROVED** [review]

File: src/ess/amor/data.py
Comment (jl-wynen) [inline src/ess/amor/data.py:1]: This is not part of a copier update. Are you doing this here to test the setup? I would like this to be part of a separate PR.

**Review by jl-wynen — CHANGES_REQUESTED** [review]

**Comment by jokasimr** [comment]
The environment variables that are set from secrets are not available in the CI runs here for some reason.

I'll experiment more with this in a new repo.
